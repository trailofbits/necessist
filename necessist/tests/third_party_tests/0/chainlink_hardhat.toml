url = "https://github.com/smartcontractkit/chainlink"
rev = "v2.18.0"
subdir = "contracts"
framework = "hardhat"
parsing_only = true

[config]
ignored_functions = ["bigNumEquals", "evmRevert"]
