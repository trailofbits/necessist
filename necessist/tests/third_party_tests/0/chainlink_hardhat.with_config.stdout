Error: Failed to walk "$DIR/contracts"

Caused by:
    0: IO error for operation on $DIR/contracts/test: No such file or directory (os error 2)
    1: No such file or directory (os error 2)
